docs/exploration-and-organization/content-verification.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Content verification
+---
+
+# Content verification
+
+{% include plans-blockquote.html feature="Content verification" %}
+
+![Verified icon](./images/verified-icon.png)
+
+Admins can verify items to let others know that the content is accurate and up to date with the correct data source and query.
+
+## Benefits of verifying an item
+
+- Verified items get a fancy blue checkmark next to them.
+- They're more likely to show up higher in search suggestions and results.
```

Comment:
I think we don't need "more likely" here because we're only making a comparative statement, like we're not saying that it's always on top, but just that they're higher (which is always true)..I don't have a strong opinion though, just think we can use fewer words (and yes I realize the irony of this statement after I typed a whole paragraph)